spec/models/rss_feed/rss_feed_spec.rb

```diff
@@ -3,45 +3,6 @@
 
   before(:each) { Kernel.silence_warnings { RssFeed.const_set(:YML_DIR, Y_DIR) } }
 
-  context "with 2 hosts" do
-    before(:each) do
-      @host1 = FactoryGirl.create(:host,           :created_on => Time.utc(2013, 1, 1, 0, 0, 0))
-      @host2 = FactoryGirl.create(:host_microsoft, :created_on => @host1.created_on + 1.second)
-    end
-
-    it "#generate 2 hosts in newest_hosts rss" do
```

Comment:
just experimenting :/ 

 I don't what is happening here and why RssFeed is not model here https://travis-ci.org/ManageIQ/manageiq/jobs/205791463#L527 and I think that I did not make related change.

locally it is ActiveRecord